--- fiaif-1.3.0_1/src/fiaif.orig	Wed Sep 11 21:55:38 2002
+++ fiaif-1.3.0_1/src/fiaif	Wed Oct 23 15:14:35 2002
@@ -1,11 +1,13 @@
 #!/bin/bash
 #
-# FIAIF is an Intelligent firewall, version: $Revision$
-# Startup script to add firewall functionality.
+# fiaif		Intelligent firewall
 #
-# chkconfig: - 08 92
+# chkconfig:	345 08 92
 #
-# description: Automates a packet filtering firewall with iptables.
+# description:	Automates a packet filtering firewall with iptables.
+
+# FIAIF is an Intelligent firewall, version: $Revision$
+# Startup script to add firewall functionality.
 #
 # Script Author:	Anders Fugmann <afu at fugmann dot dhs dot org>
 # 
